ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-input-subtext/advanced-gas-fee-input-subtext.js

```diff
@@ -16,7 +16,7 @@ const AdvancedGasFeeInputSubtext = ({ latest, historical, feeTrend }) => {
           <LoadingHeartBeat />
           {latest}
         </span>
-        <img src={`./images/${feeTrend}-arrow.svg`} alt="feeTrend-arrow" />
+        <img src={`./images/${feeTrend}-arrow.svg`} />
```

Comment:
Removing the `alt` from <img/> because if feeTrends is initialized with `level` the icon will look broken with the alt message, without that it will appear to be a delayed loading.

Reply:
Is there any way to ensure that `feeTrend` is never `level`?

Reply:
I have reverted the trend changes in this PR and added a new PR with changes based on the discussion today https://github.com/MetaMask/metamask-extension/pull/13196